docs-examples/example-java/src/test/java/io/micronaut/rabbitmq/docs/event/MyStartedEventListener.java

```diff
@@ -0,0 +1,22 @@
+package io.micronaut.rabbitmq.docs.event;
+
+import io.micronaut.context.annotation.Requires;
+// tag::imports[]
+import io.micronaut.context.event.ApplicationEventListener;
+import io.micronaut.rabbitmq.event.RabbitConsumerStarted;
+import jakarta.inject.Singleton;
+// end::imports[]
+
+@Requires(property = "spec.name", value = "RabbitListenerEventsSpec")
+// tag::clazz[]
+@Singleton
+public class MyStartedEventListener implements ApplicationEventListener<RabbitConsumerStarted> {
+  @Override
+  public void onApplicationEvent(RabbitConsumerStarted event) {
+    System.out.println(new StringBuilder()
```

Comment:
use `LOG` instead of `println`

`private static final Logger LOG = LoggerFactory.getLogger(MyStartingEventListener.class);`